server.js

```diff
@@ -24,8 +26,9 @@ app.use('*', function(req, res) {
 // assumes runServer has run and set `server` to a server object
 let server;
 
-function runServer() {
+function runServer(callback) {
```

Comment:
Accidentally inherited from the old template when I was testing stuff out... I'll remove